rllib/algorithms/algorithm.py

```diff
@@ -262,7 +263,7 @@ class Algorithm(Trainable, AlgorithmBase):
 
     @staticmethod
     def from_checkpoint(
-        checkpoint: Union[str, Checkpoint, NewCheckpoint],
+        checkpoint: Union[str, Checkpoint, NewCheckpoint, _TrainingResult],
```

Comment:
can we change the places where it's using the direct output of save to instead take in the path? I feel like `Algorithm.restore` is the only thing that should take in a training result, since it's paired with `save`

Reply:
Makes sense, updated